.github/workflows/readme.yml

```
@@ -0,0 +1,40 @@
+name: Build README
+
+on:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: '36 8 * * *'
+    - cron: '36 14 * * *'
+    - cron: '36 21 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - uses: actions/cache@v2
+      name: Configure pip caching
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install Python dependencies
+      run: |
+        python -m pip install -r requirements.txt
+    - name: Update README
+      run: |-
+        python dynamic_readme.py
+        cat README.md
+    - name: Commit and push if README changed
+      run: |-
+        git diff
+        git config --global user.email "bot@example.com"
+        git config --global user.name "README-bot"
```

All the docs I see for doing this recommend this

```suggestion
        git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
        git config --global user.email "username@users.noreply.github.com"
```

---

seems fair

---

[Ok, i'm not good at reading](https://joht.github.io/johtizen/build/2022/01/20/github-actions-push-into-repository.html)

Please revert my suggestion.  Maybe something like `cdkbot@gmail.com` is better 😬

---

> ⓘ Note that the username in username@users.noreply.github.com needs to be replaced.